audio/audiotools/core/_julius.py

```diff
@@ -0,0 +1,714 @@
+# File under the MIT license, see https://github.com/adefossez/julius/LICENSE for details.
+# Author: adefossez, 2020
+"""
+Implementation of a FFT based 1D convolution in PaddlePaddle.
+While FFT is used in some cases for small kernel sizes, it is not the default for long ones, e.g. 512.
+This module implements efficient FFT based convolutions for such cases. A typical
+application is for evaluating FIR filters with a long receptive field, typically
+evaluated with a stride of 1.
+"""
+import math
+import typing
+from typing import Optional
+from typing import Sequence
+
+import paddle
+import paddle.nn as nn
+import paddle.nn.functional as F
+
+from .resample import sinc
+
```

Comment:
set `__all__` for all this interface, same like:
paddlespeech/s2t/modules/loss.py